src/drivers/intel/cavs/dmic.c

```diff
@@ -964,6 +981,12 @@ static int configure_registers(struct dai *dai,
 			dmic_prm[di]->pdm[i].enable_mic_a;
 	}
 
+	ret = stereo_helper(stereo, swap, di);
+	if (ret) {
```

Comment:
Functionally doesn't change anything, but better to check for ret < 0 for the error sake.

Reply:
OK, it's not increasing code size to do so.